Bottom: d96b1ce2f370d54aef9970a12320b0421c57a784
Top:    5a4d5f1457e75441ac6a7d5832d1c25c9798fb5a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-27 19:28:43 +0000

Refresh of find-c-avoid-modifying-bounds-

---

diff --git a/src/find.c b/src/find.c
index cd89fe0..3d46deb 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2281,8 +2281,8 @@ LookupLOConnectionsToPolygon (PolygonType *Polygon, Cardinal LayerGroup)
   info.layer = LayerGroup;
   /* check rats */
   if (setjmp (info.env) == 0)
-    r_search (PCB->Data->rat_tree, &search_box, NULL,
-              LOCtoPolyRat_callback, &info);
+    r_search (PCB->Data->rat_tree, &search_box,
+              NULL, LOCtoPolyRat_callback, &info);
   else
     return true;
 
@@ -2312,14 +2312,14 @@ LookupLOConnectionsToPolygon (PolygonType *Polygon, Cardinal LayerGroup)
           info.layer = layer;
           /* check all lines */
           if (setjmp (info.env) == 0)
-            r_search (LAYER_PTR (layer)->line_tree, &search_box, NULL,
-                      LOCtoPolyLine_callback, &info);
+            r_search (LAYER_PTR (layer)->line_tree, &search_box,
+                      NULL, LOCtoPolyLine_callback, &info);
           else
             return true;
           /* check all arcs */
           if (setjmp (info.env) == 0)
-            r_search (LAYER_PTR (layer)->arc_tree, &search_box, NULL,
-                      LOCtoPolyArc_callback, &info);
+            r_search (LAYER_PTR (layer)->arc_tree, &search_box,
+                      NULL, LOCtoPolyArc_callback, &info);
           else
             return true;
         }
@@ -2327,8 +2327,8 @@ LookupLOConnectionsToPolygon (PolygonType *Polygon, Cardinal LayerGroup)
         {
           info.layer = layer - max_copper_layer;
           if (setjmp (info.env) == 0)
-            r_search (PCB->Data->pad_tree, &search_box, NULL,
-                      LOCtoPolyPad_callback, &info);
+            r_search (PCB->Data->pad_tree, &search_box,
+                      NULL, LOCtoPolyPad_callback, &info);
           else
             return true;
         }
